configuration/Makefile

```diff
@@ -53,33 +53,38 @@ validate: $(KUBEVAL) generate
 
 # Use Jsonnet to generate JSON versions of K8s manifests without .json extensions,
 # then send all the JSON through xargs to be converted into YAML and given .yaml extension.
+# The goal of the sed at the end is to remove quotes from an env var in the loki config, as
+# it must be read by loki as an int and not as a string
 examples/base/manifests: examples/base/main.jsonnet vendor $(JSONNET_SRC) $(JSONNET) $(GOJSONTOYAML)
 	echo ecs "$OS"
 	-make fmt
 	-rm -rf examples/base/manifests
 	-mkdir examples/base/manifests
-	$(JSONNET) -J vendor -c -m examples/base/manifests examples/base/main.jsonnet | $(XARGS) -I{} sh -c 'cat {} | $(GOJSONTOYAML) > {}.yaml' -- {}
+	$(JSONNET) -J vendor -J lib -c -m examples/base/manifests examples/base/main.jsonnet | $(XARGS) -I{} sh -c 'cat {} | $(GOJSONTOYAML) > {}.yaml' -- {}
 	find examples/base/manifests -type f ! -name '*.yaml' -delete
+	sed -i 's/"$${LOKI_REPLICATION_FACTOR}"/$${LOKI_REPLICATION_FACTOR}/g' examples/base/manifests/observatorium/loki-config-map.yaml
```

Comment:
@periklis I had to do this hack 😞 for the replication_factor to be read by loki as an int and not as a string